tox.ini

```diff
@@ -12,8 +12,11 @@ minversion = 2.9
 description = "Scapy unit tests"
 whitelist_externals = sudo
 setenv = SCAPY_ROOT_DIR={env:PWD}
+passenv = SCAPY_USE_PCAPDNET
 deps = mock
        cryptography
+       pcapy==0.11.3
```

Comment:
How about using > instead of == (not sure if supported in tox)
Same above in the warning, it should be saying `Please use pcapy 0.11.3+`

Reply:
Good idea!